CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+How to contribute
+=================
+
+We want Iris and its sister packages (such as iris-grib) to be driven by
+the community - your contributions are invaluable to us! This page lists
+the guidelines for contributors which will help ease the process of
+getting your hard work accepted into iris-grib, and shared back to the world.
+
+Getting started
+---------------
+
+1. If you've not already got one, sign up for a
+   [GitHub account](https://github.com/signup/free).
+1. Fork the iris-grib repository, create your new fix/feature branch, and
+   start commiting code.
+ - The main
+   [Iris development guide](http://scitools.org.uk/iris/docs/latest/developers_guide/gitwash/git_development.html)
+   has more detail.
+1. Remember to add appropriate documentation and tests to supplement any new or changed functionality.
+
+
+Submitting changes
+------------------
+
+1. Read and sign the Contributor Licence Agreement (CLA).
+ - See our [governance page](http://scitools.org.uk/governance.html)
+   for the CLA and what to do with it.
+1. Push your branch to your fork of iris-grib.
```

Comment:
Agreed, wouldn't it render equally well if we used, say, the actual number you want... A lot of people will read this a plain text and it looks weird like this.